app/code/Inchoo/Sample03/view/frontend/templates/hello-world.phtml

```diff
@@ -0,0 +1,9 @@
+<?php
+/** @var \Magento\Framework\View\Element\Template $block */
+/** @var \Magento\Framework\Escaper $escaper */
+?>
+
+<h3>Template block</h3>
```

Comment:
code check ovdje nebaca error?

Reply:
U svakom .phtml-u mi baca error $ecscaper is undefined.

Reply:
ne bi trebao ako imaš komentar tipa:
/** @var $escaper Magento\Framework\Escaper */

generalno ovakve stvari riješavamo unutar modula da prolaze code check, a u temi ako smo uzeli template direktno od magenta nesređujemo ga za code check u ovom smislu jer bude pun takvih stvari.

ovo sa komentarom bi ti trebalo radit, a gdje ne escapeaš string trebaš komentar ovako stavit i wrappat string u funkciju za prijevod da ga zna pokupit ukoliko prijevod postoji:

```suggestion
<h3><?= /** @noEscape */ __('Template block') ?></h3>
```